features/prefetch.yml

```diff
@@ -0,0 +1,7 @@
+name: Prefetch
+description: The `Sec-Purpose` HTTP header with a value of `prefetch` signals that a resource can be requested in anticipation of a near future navigation, such as a page linked in search results or a link that a user has hovered over.
+spec: https://wicg.github.io/nav-speculation/prefetch.html
+compat_features:
+  - api.PerformanceResourceTiming.deliveryType.navigational-prefetch
```

Comment:
I'm not sure what to do with this key. It could belong with `resource-timing` or `link-rel-prefetch`. @Elchi3 do you have an opinion about it?

Reply:
I wrote rationale for prefetch as a separate feature in comment #0, but don't feel strongly about it. There was no easy out in this stack of vaguely-intersecting keys.

Reply:
Yeah after removing the header keys, this doesn't make sense as a single key feature.

In previous situations like this, we've moved the key into the feature the key depends on, which would be `link-rel-prefetch` in this case.

Reply:
Moved all these keys into `link-rel-prefetch` for now.